docs/iris/src/whatsnew/contributions_v2.0.0/incompatiblechange_2017-Oct-19_remove_regridded.txt

```diff
@@ -0,0 +1 @@
+* The deprecated method ```iris.cube.Cube.regridded``` has been removed from the module :mod:`iris.cube`.
```

Comment:
Normally you'd say ```has been removed from the Cube```, no? The module is a little bit academic in this regard.

Reply:
@pelson wherever possible we've been using Sphinx doc links for things that still exist, so I'm happy with this as it is, FWIW.

Reply:
I don't know, would you?  I never normally say that, but if you say that's best then I believe you and I'll change my wording.

Reply:
@dkillick @pelson How about ```has been removed from :mod:`iris.cube` ```?